package.json

```diff
@@ -14,11 +14,13 @@
     "storybook": "start-storybook -s ./public -p 6006",
     "storybook:build": "build-storybook -s ./public",
     "chromatic": "chromatic --exit-zero-on-changes --skip 'dependabot/**'",
-    "generate": "yarn plop --plopfile generators/plopfile.js"
+    "generate": "yarn plop --plopfile generators/plopfile.js",
+    "prepare": "husky install",
+    "lint-staged": "lint-staged"
   },
   "husky": {
     "hooks": {
-      "pre-commit": "lint-staged",
+      "pre-commit": "yarn run lint-staged",
       "pre-push": "yarn test"
     }
```

Comment:
This key is no longer valid. The new API from Husky uses the `.husk/filename` to match hooks.